WSL/release-notes.md

```diff
@@ -10,6 +10,15 @@ ms.localizationpriority: high
 
 # Release Notes for Windows Subsystem for Linux
 
+## Build 20190
+For general Windows information on build 20190 visit the [Windows blog](https://blogs.windows.com/windowsexperience/2020/08/12/announcing-windows-10-insider-preview-build-20190/).
+
+* Fix bug preventing WSL1 instances from launching [GH 5633]
+* Fix hang when redirecting Windows process output [GH 5648]
+* Add %userprofile%\\.wslconfig option to control the VM idle timeout (wsl2.vmIdleTimeout=<time_in_ms>)
```

Comment:
```suggestion
* Add `%userprofile%\\.wslconfig` option to control the VM idle timeout (`wsl2.vmIdleTimeout=<time_in_ms>`)
```

do we support/should we use code blocks for these?

Reply:
Good call -- yes -- I think code blocks should be used. Improves readability.